orientdb/src/test/java/com/yahoo/ycsb/db/OrientDBClientTest.java

```diff
@@ -0,0 +1,236 @@
+/**
+ * Copyright (c) 2015 YCSB contributors. All rights reserved.
```

Comment:
I actually wrote this in 2015, but the PR came in 2016, does one matter over the other?